query07.sql

```diff
@@ -3,4 +3,11 @@
 */
 
 -- Enter your SQL query here
-select ...
+select count(*), '2019' as year
+from indego_trips_2019_q2
+where extract(day from end_time :: date) != extract (day from start_time :: date)
+union
+select count(*), '2020' as year
+from public.indego_trips_2020_q2
+where extract(day from to_date(start_time, 'MM/DD/YYYY HH24/MI/SS')) != extract(day from to_date(end_time, 'MM/DD/YYYY HH24/MI/SS'))
```

Comment:
This actually over-counts. Even though the max trip `duration` is entered as `1440`, some trips start on one day and end _after_ the next day. The solution I used is [here](https://github.com/musa-509-fall-2021/assignment-01-intro-to-geographic-queries/blob/answers/query07.sql#L16-L25) (or a more compact equivalent solution [here](https://github.com/musa-509-fall-2021/assignment-01-intro-to-geographic-queries/blob/answers/query07.sql#L40-L42))